src/views/Farms/Farms.tsx

```diff
@@ -232,6 +235,10 @@ const Farms: React.FC = ({ children }) => {
           return orderBy(farms, (farm: FarmWithStakedValue) => Number(farm.liquidity), 'desc')
         case 'latest':
           return orderBy(farms, (farm: FarmWithStakedValue) => Number(farm.pid), 'desc')
+        case 'hot': {
```

Comment:
could we do this logic in the `contants/farms.ts` file ?

Reply:
assign to @ChefMomota. Let's make it static

Reply:
👍 I will close this pr